packages/material-ui/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -110,6 +110,7 @@ const Breadcrumbs = React.forwardRef(function Breadcrumbs(props, ref) {
 
   return (
     <Typography
+      aria-label="Breadcrumb"
```

Comment:
Why do we need this label for this feature?

Reply:
I just try to follow wai-aria [example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/breadcrumb/index.html).
I will remove it, no problem

Reply:
A good opportunity to settle on this matter:
https://github.com/mui-org/material-ui/blob/c9745319fab39ceee5390de7b9bd8db2f830221a/packages/material-ui-lab/src/Pagination/Pagination.js#L58

Reply:
I've removed, did not notice it in [docs](http://localhost:3000/components/breadcrumbs/#accessibility)
My bad:)